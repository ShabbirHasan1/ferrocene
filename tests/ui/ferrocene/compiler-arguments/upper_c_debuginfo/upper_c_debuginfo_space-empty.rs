// check-fail
// compile-flags: -C debuginfo

fn main() {}

// ferrocene-annotations: um_rustc_C_debuginfo
