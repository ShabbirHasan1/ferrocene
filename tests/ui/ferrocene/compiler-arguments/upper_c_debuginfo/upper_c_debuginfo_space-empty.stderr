error: codegen option `debuginfo` requires a number (C debuginfo=<value>)

